doc/man3/SSL_get_value_uint.pod

```diff
@@ -162,18 +178,82 @@ of QUIC stream creation flow control.
 Can be queried using the convenience macro
 SSL_get_quic_stream_bidi_remote_avail().
 
-=item B<SSL_VALUE_QUIC_STREAM_UNI_REMOTE_AVAIL>
+=item B<SSL_VALUE_QUIC_STREAM_UNI_REMOTE_AVAIL> (connection object)
 
 As above, but provides the number of unidirectional, remotely-initiated streams
 available to be created (but not yet created).
 
 Can be queried using the convenience macro
 SSL_get_quic_stream_uni_remote_avail().
 
+=item B<SSL_VALUE_EVENT_HANDLING_MODE> (connection or stream object)
+
+Generic value. This is an integer value which takes one of the following values,
+and determines the event handling mode in use:
+
+=over 4
+
+=item B<SSL_VALUE_EVENT_HANDLING_MODE_INHERIT>
+
+When set, the event handling mode used is inherited from the value set on the
+parent connection (for a stream), or, for a connection, defaults to the implicit
+event handling model.
+
+When a new connection is created, or a new stream is created or accepted, it
+defaults to this setting.
+
+=item B<SSL_VALUE_EVENT_HANDLING_MODE_IMPLICIT> (Implicit event handling)
+
+If set to this value (the default), the implicit event handling model is used.
```

Comment:
This is a bit confusing. It says this value is the default. But just above we say that SSL_VALUE_EVENT_HANDLING_MODE_INHERIT is the default for new connections.